cypress/integration/podcast.js

```diff
@@ -0,0 +1,125 @@
+const TWITTER_SHARE_URL = `https://twitter.com/intent/tweet?url=${PROD_PODCAST_URL}&text=This%20is%20the%20MongoDB%20Podcast`;
+
+const SOCIAL_URLS = [LINKEDIN_SHARE_URL, TWITTER_SHARE_URL, FACEBOOK_SHARE_URL];
+
+const BREADCRUMBS = ['Home', 'Learn', 'Podcast'];
+
+describe('Sample Podcast Page', () => {
+    it('should have a descriptive header', () => {
+        cy.visit(PODCAST_SLUG).then(() => {
+            cy.get('header').within(() => {
+                cy.contains(PODCAST_TITLE);
+                // Check pubdate
+                cy.contains(/Published: Feb 25, 2020|Published: 25 Feb 2020/);
+            });
+        });
+    });
+
+    it('should have links to share content on social media', () => {
+        cy.get('[data-test="article-share-links"]').within(() => {
+            cy.get('a').each((link, index) => {
+                if (index !== 0) {
+                    const url = SOCIAL_URLS[index - 1];
+                    cy.wrap(link).should('have.prop', 'href', url);
+                }
+            });
+        });
+    });
+
+    it('should have h1 title and description', () => {
+        cy.get('h1').contains(PODCAST_TITLE);
+    });
+
+    it('should have background image', () => {
+        cy.get('header > div')
+            .should('have.css', 'background-image')
+            .and('match', new RegExp(IMAGE));
+    });
+
+    it('should have proper SEO, open graph, and twitter tags', () => {
+        // Check title exists
+        cy.title().should('eq', PODCAST_TITLE).end();
+
+        // Check og tags
+        cy.checkMetaContentProperty('property="og:type"', 'article');
+        cy.checkMetaContentProperty('property="og:title"', PODCAST_TITLE);
+        cy.checkMetaContentProperty(
+            'property="og:url"',
+            `${PROD_PODCAST_URL}/`
+        );
+        cy.checkCanonicalUrlValue(`${PROD_PODCAST_URL}/`);
+        // An og:description exists, so we should populate the tag with it
+        cy.checkMetaContentProperty(
+            'property="og:description"',
+            PODCAST_DESCRIPTION
+        );
+        cy.checkMetaContentProperty('name="description"', PODCAST_DESCRIPTION);
+        cy.checkMetaContentProperty('property="og:image"', IMAGE);
+
+        // Check Twitter tags
+        cy.checkMetaContentProperty('name="twitter:card"', 'summary');
+        cy.checkMetaContentProperty('name="twitter:site"', '@mongodb');
+        cy.checkMetaContentProperty('property="twitter:title"', PODCAST_TITLE);
+        cy.checkMetaContentProperty(
+            'property="twitter:description"',
+            PODCAST_DESCRIPTION
+        );
+        cy.checkMetaContentProperty('name="twitter:image"', IMAGE);
+    });
+
+    it('should have breadcrumb', () => {
+        cy.get('header a').each((el, index) => {
+            cy.wrap(el).contains(BREADCRUMBS[index]);
+        });
+    });
+
+    it('should have audio player', () => {
```

Comment:
Perhaps we could use the Cypress `describe` nested here to group audio player tests together. What do you think? Then we can use `before` perhaps for the `wait` if we are concerned about it?

Reply:
Nice idea!